week7/rlawngP91/mission/src/pages/cart/Cart.jsx

```diff
@@ -0,0 +1,73 @@
+import styles from './Cart.module.css';
+import CartMenuItem from '../../components/menuItem/CartMenuItem';
+import { exclamation } from '../../assets';
+import useCartStore from '../../store/cartStore';
+
+const Cart = () => {
+
+    const store = useCartStore((state) => state.store);
+    const menus = useCartStore((state) => state.menus);
+    
+    // 총 주문금액 계산
+    const subtotal = menus.reduce((sum, menu) => sum + menu.price, 0);
+    const deliveryFee = store?.deliveryFee;
+    const total = subtotal + deliveryFee;
+
```

Comment:
총 주문 금액 계산하는 로직을 이렇게 분리하는 것도 좋을 것 같아요

1) 추후에 주문 금액 계산하는 로직에 수정이 필요할 때 `getTotalPrice` 함수만 찾아가면 됨
2) 기존 코드에서는 12-14 라인이 총 주문 금액을 계산하는 코드라는 것을 드러내기 위해 주석으로 표현되어 있는데 이렇게 분리하면 함수명 자체에서 그 의미를 파악할 수 있어요. 이렇게 의미 파악이 쉬워질 수록 다른 작업자의 인지적 부담도 덜어지고 전체적인 코드의 가독성도 좋아져용

```suggestion
const getTotalPrice = ({ menus, deliveryFee }) => {
    const menusPrice = menus.reduce((sum, menu) => sum + menu.price, 0);
    const totalPrice = menusPrice + deliveryFee;
    return { totalPrice, menusPrice }
}

const { totalPrice,  menusPrice } = getTotalPrice({ menus, deliveryFee: store?.deliveryFee ?? 0 });
```